src/styles/index.scss

```diff
@@ -1,3 +1,300 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property is usually better to be applied to all elements in your CSS. You can do this with `*, *::before, *::after { box-sizing: border-box; }`. This ensures that padding does not affect the final computed width of an element.